### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by pavpanchekha**

### CODE REVIEW

File: book/forms.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ HTML forms have a couple of moving parts.
 
 First, in HTML there is a `form` element, which contains `input`
 elements,[^or-others] which in turn can be edited by the user. So a
-form might be written like this:
+form might be written like this (see results in Figure 1):
```
Comment (pavpanchekha) [inline book/forms.md:22]: Minor, just moving this up a few lines.

File: book/forms.md [hunk 2]
```diff
@@ -239,10 +237,10 @@ resolve the block-mixed-with-inline-siblings problem (see
...
 children, yet must have inline layout or else they won't draw correctly.
 Likewise, a `<button>` does have children, but they are treated
-specially.^[This situation is specific to these elements in our browser, but
-only because they are the only elements with special painting
-behavior within an inline context. These are also two examples of
-[atomic inlines](https://www.w3.org/TR/CSS2/visuren.html#inline-boxes).]
+specially.^[These are two examples of
+[atomic
+inlines](https://www.w3.org/TR/CSS2/visuren.html#inline-boxes)---the
+only two examples in our browser so far.]
```
Comment (pavpanchekha) [inline book/forms.md:243]: I think the original was hard to parse.

File: book/forms.md [hunk 3]
```diff
@@ -522,13 +520,10 @@ class Tab:
...
 ```
 
-Note that we have to un-focus[^blur] the currently focused element,
+Note that we have to un-focus the currently focused element,
 lest it keep drawing its cursor. Anyway, now we can draw a cursor if
 an `input` element is focused:
 
-[^blur]: Un-focusing is called "blurring", which is funny but can
-    sometimes lead to confusion.
-
```
Comment (pavpanchekha) [inline book/forms.md:531]: This is already covered earlier.

File: book/forms.md (hunk 1 above)
Comment (chrishtr) [reply book/forms.md:22]: Not critical IMO

File: book/forms.md (hunk 2 above)
Comment (chrishtr) [reply book/forms.md:243]: I think the original text is ok?

File: book/forms.md [hunk 4]
```diff
@@ -1194,7 +1189,7 @@ Each topic should have its own URL and its own list of messages. So,
 for example, `/cooking` should be a page of posts (about cooking) and
 comments submitted through the form on that page should only show up
 when you go to `/cooking`, not when you go to `/cars`. Make the home
-page, from `/`, list the available topics with a link to each topic's
```
Comment (chrishtr) [inline book/forms.md:1197]: why?

File: book/forms.md (hunk 2 above)
Comment (pavpanchekha) [reply book/forms.md:243]: Not critical but I do think it’s an improvement. Perhaps not worth it.

File: book/forms.md (hunk 4 above)
Comment (pavpanchekha) [reply book/forms.md:1197]: When reading I couldn’t understand what “from /“ means, and I guess it doesn’t seem too important to specify the URLs, the exercise is pretty open ended.

Comment (chrishtr) [reply book/forms.md:1197]: I think it'd be better to revert this edit.

File: book/forms.md (hunk 2 above)
Comment (chrishtr) [reply book/forms.md:243]: The new text also loses information..

File: book/forms.md (hunk 4 above)
Comment (pavpanchekha) [reply book/forms.md:1197]: Ok, new suggestion: change `from /` to `at /`

File: book/forms.md (hunk 2 above)
Comment (pavpanchekha) [reply book/forms.md]: Reverted.
